tests/ctl/core/test_privacy_declaration.py

```diff
@@ -2,24 +2,30 @@
 
 
 class TestPrivacyDeclaration:
-    def test_privacy_declaration_datasets(
-        self,
-        privacy_declaration_with_dataset_references: PrivacyDeclaration,
-    ) -> None:
-        assert len(privacy_declaration_with_dataset_references.datasets) == 1
+
+    def test_privacy_declaration_dataset_data_categories(
+        self, privacy_declaration_with_multiple_dataset_references
+    ):
+        assert set(
+            privacy_declaration_with_multiple_dataset_references.dataset_data_categories
+        ) == {
+            "user.behavior",
+            "user.unique_id",
+            "user.contact.address.street",
+        }
```

Comment:
Check out the `privacy_declaration_with_multiple_dataset_references` fixture to see the different levels of nesting for the data categories